agent/internal/containers.go

```diff
@@ -222,6 +242,45 @@ func gpuDeviceRequests(cont cproto.Container) []dcontainer.DeviceRequest {
 	}
 }
 
+func injectRocmDeviceRequests(cont cproto.Container, hostConfig *dcontainer.HostConfig) error {
+	uuids := cont.DeviceUUIDsByType(device.ROCM)
+
+	if len(uuids) == 0 {
+		return errors.New("no rocm device uuids")
+	}
+
+	hostConfig.SecurityOpt = append(
+		hostConfig.SecurityOpt, "seccomp=unconfined")
+	hostConfig.GroupAdd = append(hostConfig.GroupAdd, "video")
+	mappedDevices := []string{"/dev/kfd"}
+
+	for _, uuid := range uuids {
+		rocmDevice := getRocmDeviceByUUID(uuid)
+		devPaths := []string{
+			fmt.Sprintf("/dev/dri/by-path/pci-%s-card", strings.ToLower(rocmDevice.PCIBus)),
+			fmt.Sprintf("/dev/dri/by-path/pci-%s-render", strings.ToLower(rocmDevice.PCIBus)),
+		}
+		for _, symlink := range devPaths {
+			resolved, err := filepath.EvalSymlinks(symlink)
```

Comment:
are there some tools that may look where the symlink was, not just the absolute paths (since I guess we're only mounting those)?